app/graphql/types/objects/question_type.rb

```diff
@@ -1,14 +1,29 @@
 module Types
-	module Objects
-		class QuestionType < BaseObject
-			graphql_name "BaseQuestionType"
-			field :id, 						Int, "ID of the question", null: false
-			field :is_optional,		Boolean, nil, null: false
-		  field :question_type, String, nil, null: true
-		  field :question_text, String, nil, null: true
-		  field :parent, 				Types::Objects::QuestionType, nil, null: true
-		  field :sub_questions, [Types::Objects::QuestionType], nil, null: true
-		  field :supports_other, Boolean, nil, null: false
-		end
-	end
-end
\ No newline at end of file
+	 module Objects
+ 		 class QuestionType < BaseObject
+  			graphql_name 'BaseQuestionType'
+  			field :id,	Int, 'ID of the question', null: false
+  			field :is_optional,	Boolean, nil, null: false
+  		  field :question_type, String, nil, null: true
+				field :question_text, String, nil, null: true
+  		  field :english_question_text, String, nil, null: true
```

Comment:
@austinjb32 - I appreciate you trying to make it even more clear that it is the "english question text", but we can keep it as `question_text` because english is well understood as the default on the platform.